Makefile

```
@@ -1,15 +1,22 @@
 # For Linux and MacOS users only and assumes brew is already installed
 TECH_MARKDOWN_FILE := doc/technical-overview.md
-PDF_OUTFILE := dds_web/static/dds-technical-overview.pdf
+TECH_OVERVIEW_PDF := dds_web/static/dds-technical-overview.pdf
+TROUBLE_MARKDOWN_FILE := doc/troubleshooting.md
+TROUBLE_PDF := dds_web/static/dds-troubleshooting.pdf
 
 install_pandoc:
-	brew install pandoc
+	brew install pandoc texlive
```

```suggestion
	brew install pandoc librsvg
```
Installing `texlive` did not work for me, I needed to install the base Mactex package; maybe that's just for MacOS but not sure


´librsvg´ should be installed via brew and not tlmgr

---

Interesting. Let me investigate the texlive thing - I'm also on MacOS.